integrations/access/slack/bot.go

```diff
@@ -45,21 +45,30 @@ type Bot struct {
 }
 
 // onAfterResponseSlack resty error function for Slack
-func onAfterResponseSlack(_ *resty.Client, resp *resty.Response) error {
-	if !resp.IsSuccess() {
-		return trace.Errorf("slack api returned unexpected code %v", resp.StatusCode())
-	}
+func onAfterResponseSlack(sink common.StatusSink) func(_ *resty.Client, resp *resty.Response) error {
+	return func(_ *resty.Client, resp *resty.Response) error {
+		var status types.PluginStatus
+		defer func() {
+			common.TryEmitStatus(context.TODO(), sink, status)
```

Comment:
Don't context.TODO() unless you plan a follow up. Use context.Background() instead.

(If you do plan a follow up, leave a good old TODO behind too.)

Reply:
Will change to `Background()` for now here, as there's no other context in scope.